src/app/_components/Avatar.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
+import Image from 'next/image'
+
+import { type DigestArticleData } from '@/types/digestTypes'
+import type { ImageProps } from '@/types/sharedProps/imageType'
+
+type ImagePropsWithoutAlt = Omit<ImageProps, 'alt'>
+
+export function Avatar({ authors }: { authors: DigestArticleData['authors'] }) {
+  const imageSize = 32
+
+  if (!authors || authors.length === 0) {
```

Comment:
Nit: `authors` is mandatory so checking for its existence is unnecessary?

```suggestion
  if (authors.length === 0) {
```